es.hjson

```diff
@@ -432,56 +432,56 @@
+  "joinverify.preverify.error.notAvailable": "<b>JoinVerify no está activado en este momento. Debe estar activado antes de poder usar PreVerify:</b>"
 
   "joinverify.preverify.steps": {
-    "one": "✴️ <b>1. Reenvíe el mensaje anterior (respondido a)</b> a un canal público donde %s es un administrador.\n<b>2. Toque o haga clic con el botón derecho en el mensaje reenviado y <b>Copiar enlace</b>\n3. Envíe el enlace público copiado a este chat</b>.\n\nEl enlace debería verse así:\nhttps://t.me/CalsiBotDev/99999"
+    "one": "✴️ <b>1. Reenvíe el mensaje anterior (respondido a)</b> a un canal público donde %s es un administrador.\n<b>2. Toque o haga clic con el botón derecho en el mensaje reenviado y seleccione <b>Copiar enlace</b>\n3. Envíe el enlace público copiado a este grupo o chat</b>.\n\nEl enlace debería ser algo parecido a esto:\nhttps://t.me/CalsiBotDev/99999"
     "oneError.noVerificationString": "❌ <b>No se pudo encontrar la cadena de verificación requerida en la publicación. Asegúrese de que la publicación a la que se vinculó sea el mensaje de verificación de propiedad que reenvió.</b>"
     "oneError.urlFormat": "❌ <b>El enlace que envió no se ve bien. Debería verse así:</b>\nhttps://t.me/CalsiBotDev/99999"
-    "oneError.targetIsNotChannel": "❌ <b>El enlace que ha proporcionado es para un Grupo público.\n\nPreVerify está diseñado para usarse solo con canales públicos.</b>"
-    "oneError.cantSendPost": "❌ <b>No podría enviar el mensaje, asegúrese de que %s sea un administrador con suficientes permisos en %s.</b>%s"
-    "oneSuccess": "✅ <b>¡Ok, publicado con éxito!</b>\n\nVer https://t.me/%s/%s para la publicación!\n\n<b>proteja a su grupo, debe revocar todos los enlaces de invitación anteriores. Puede hacerlo mediante:</b>\n1. Revocando su Invite Link personal\n2. Degradar y luego volver a promocionar a todos sus administradores (excepto CalsiBot). Esto revocará cualquier enlace de invitación que hayan compartido."
+    "oneError.targetIsNotChannel": "❌ <b>El enlace que ha proporcionado es para un Grupo público.\n\nPreVerify está definido para usarse solo con canales públicos.</b>"
+    "oneError.cantSendPost": "❌ <b>No he podido enviar el mensaje. Asegúrese de que %s sea un administrador con suficientes permisos en %s.</b>%s"
+    "oneSuccess": "✅ <b>¡Ok, publicado con éxito!</b>\n\nVer https://t.me/%s/%s para la publicación!\n\n<b>Para proteger completamente su grupo, debe revocar todos los enlaces de invitación anteriores. Puede hacerlo de la siguiente manera:</b>\n1. Revoca tu enlace de invitación personal\n2. Remueve y vuelve a promover a tus administradores nuevamente (excepto CalsiBot). Esto revocará cualquier enlace de invitación que hayan compartido."
   }
 
   "joinverify.preverify.imageText": {
     "title": "Para unirse al grupo:"
     "stepOne": "Toque <span>Verificar,</span> luego <span>Inicio</span>"
-    "stepTwo": "<span>Pase</span> la prueba"
+    "stepTwo": "<span>Pase</span> la verificación humana"

```

Comment:
Space is very limited here. We must find a way to make this shorter.

If we can make the `stepOne` shorter also this would be amazing.

This text will be used to automatically create an image, like this:
![image](https://user-images.githubusercontent.com/55633371/140170596-bea0ee20-882f-486d-b7dd-87ae02e6c422.png)